flink-eval/src/main/scala/io/concord/eval/LineParser.scala

```diff
@@ -0,0 +1,21 @@
+package io.concord.eval
+
+case class LogLine(
+  tsSecs: Int,
+  fullLine: String,
+  message: String
+)
+
+object LineParser {
+  // Thanks Alex
+  private[this] val lineRegex =
+    """-\s(\d+)\s\d+\.\d+\.\d+\s\w+\s\w+\s\d+\s\d+:\d+:\d+\s\w+@\w+(.*)$""".r
```

Comment:
this is a bugsie, will push a new regex soon on the cpp and here ! making a note